test/dlc_tests/benchmark/sagemaker/tensorflow/training/test_performance_tensorflow_sm_training.py

```diff
@@ -11,6 +11,9 @@
 from test.test_utils import BENCHMARK_RESULTS_S3_BUCKET, LOGGER
 
 
+@pytest.mark.integration("sagemaker performance benchmark with imagenet dataset")
```

Comment:
Wouldn't it be better to make the marker more generic by putting in something like "sm_perf" or something like that?

Also, do you think it would be good to have information about the dataset used by a test as another pytest marker? That would provide a good dimension to analyze the tests we use.

Reply:
The string inside is not a marker, but an argument to the "integration" marker.

Good idea re: dataset fixture. I thought a similar thing, though it was not included in the AC for this task. I will follow up in a separate PR to add that info

Reply:
I think "sm_perf" wouldn't provide much value as an integration since that info is included in the filepath. To extend past the test report generation doc, if someone wants to run sm perf tests manually, they would know how to do so without a "sm_perf" fixture due to the directory structure. I'd be OK with shortening this fixture to just be "imagenet" or removing altogether -- want to ensure that whatever we add is providing value to the report.

Reply:
i changed this to "imagenet dataset"